docs/source/developer_zone/dev_environment.rst

```diff
@@ -127,9 +127,20 @@ Install development dependencies
 
 .. code:: bash
 
-    micromamba create -n mamba -c conda-forge -f dev/environment.yml
+    micromamba create -n mamba -c conda-forge -f dev/environment-dev.yml
     micromamba activate -n mamba
 
+.. note::
+
+    You need to have ccache installed on your system.
+    You can either install it via your package manager or via conda-forge
+    in this development environment with:
+
+    .. code:: bash
+
+        micromamba install -n mamba -c conda-forge ccache
+
+
```

Comment:
I think we can completely remove this part as `ccache` is related to the provided Preset (so the note below is enough).